functions/APIs/deals.js

```diff
@@ -0,0 +1,115 @@
+const { db } = require('../util/admin');
+
+exports.getAllDeals = (request, response) => {
```

Comment:
While most of this is pretty self-documenting, it might be worthwhile to add some documentation for what all the request and response parameters are.  Otherwise it's not really clear what "farmlinkContactKey" represents, or what format we should expect "deliveryDate" to be in, or what "bill" means.

Reply:
I added documentation for the data types of the data parameters.  Should I add documentation for the meaning of some of the properties?

Reply:
Yeah, certainly not for all of them (it's pretty obvious what "farmId" means, for example), but for some of the more ambiguous ones it would be very helpful to both me and potentially your podmates.